build_library/qemu_template.sh

```diff
@@ -111,7 +111,7 @@ write_ssh_keys() {
 
 
 if [ -z "${CONFIG_IMAGE}" ]; then
-    CONFIG_DRIVE=$(mktemp -t -d flatcar-configdrive.XXXXXXXXXX)
+    CONFIG_DRIVE=$(mktemp -d -t flatcar-configdrive)
```

Comment:
Removing the Xs breaks GNU mktemp. Does this work on MacOS?

```suggestion
    TEMPSUFFIX=""
    if mktemp --version | grep -q GNU; then
      TEMPSUFFIX=".XXXXXXXXXX"
    fi
    CONFIG_DRIVE=$(mktemp -d -t "flatcar-configdrive${TEMPSUFFIX}")
```

Reply:
Yes it does, but simply adds the random part at the end like "flatcar-configdrive.XXXXXXXXXX.FkrR1lVh".

Reply:
With my suggestion or with the way it was before?

Reply:
How about we leave the X's and just accept that the way the name is constructed is slightly different. At the end of the day, we only need it to be slightly unique on both platform, no reason to overcomplicate things.

Reply:
Ok, updated my suggestion

Reply:
I tried to make it even simpler by removing the -t and the template.